### ISSUE TITLE
No example page of canary blue/green deployment for ECS, AWS Lambda 

### ISSUE THREAD

**Issue by kentakozuka** [pipe-cd/pipecd#4467]
https://pipecd.dev/docs/user-guide/examples/

**Comment by t-kikuc** [comment]
Let me fill in the gap of context.

## What we want

- Example pages of Canary, Blue/Green deployment for ECS, Lambda


## Why we want

- It's tough for users to understand what happens while progressive delivery for ECS/Lambda



---

For ECS, I recommend referring to my article:
https://dev.to/t-kikuc/ecs-external-deployment-taskset-complete-guide-21dl#method-b-using-three-tasksets

**Comment by ubermensch19** [comment]
@t-kikuc , Can I work on it? or its not needed?

**Comment by t-kikuc** [comment]
@priyanshuharshbodhi1 
thank you, not needed for now.
please work on it after releasing the ECS/Lambda plugins

**Comment by abdollahShahid** [comment]
@t-kikuc Hi, I would like to contribute. I noticed the earlier note that this may depend on the ECS/Lambda plugin status, so I wanted to check before starting. If this issue is actionable now, I’d be happy to work on it.

**Comment by riahtgl** [comment]
Hi @abdollahShahid, are you still interested in working on this?

**Comment by abdollahShahid** [comment]
> Hi @abdollahShahid, are you still interested in working on this?

sure

**Comment by riahtgl** [comment]
Alright, I've assigned the issue to you. Kindly proceed. Please let us know of any updates as you progress. Thank you for your time and effort.

> > Hi [@abdollahShahid](https://github.com/abdollahShahid), are you still interested in working on this?
> 
> sure

**Comment by abdollahShahid** [comment]
> Alright, I've assigned the issue to you. Kindly proceed. Please let us know of any updates as you progress. Thank you for your time and effort.
> 
> > > Hi [@abdollahShahid](https://github.com/abdollahShahid), are you still interested in working on this?
> > 
> > sure
> 
> 

thanks! will do this weekend.

**Comment by Prachi-Gupta2808** [comment]
hello @riahtgl , if no updates till now from @abdollahShahid , can I work on this issue?

**Comment by abdollahShahid** [comment]
> hello @riahtgl , if no updates till now from @abdollahShahid , can I work on this issue?

go ahead

**Comment by Prachi-Gupta2808** [comment]
I am working on this and would raise a PR soon.

**Comment by Prachi-Gupta2808** [comment]
I am dividing the work into 2 PRs for the ease.
